docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,414 @@
-* `$-` refers to the input stream.
+There are two ways to get the same result. First, we can use a `pipe` to retrieve the team. Then we use a `temporary variable` to retrieve the same team.
+
+Enter the following statement with a `pipe`:
+
+```
+GO FROM 100 OVER follow YIELD follow._dst AS id | GO FROM $-.id OVER serve YIELD $$.team.name AS Team;
+```
+
+The following information is returned.
+
+```
+===========
+| Team    |
+===========
+| Nuggets |
+-----------
+```
+
+Enter the following statement with a `temporary variable`:
+
+```
+$var=GO FROM 100 OVER follow YIELD follow._dst AS id; GO FROM $var.id OVER serve YIELD $$.team.name AS Team;
+```
 
-The second approach adopts a user-defined variable `$var`. The scope of this variable is within the compound statement.
+The following information is returned.
 
-## Tips
+```
+===========
+| Team    |
+===========
+| Nuggets |
+-----------
+```
 
-In practice, it often takes several retries to find a proper graph schema during testing. You can put all the DDL statements in a file as follows.
+**Note**:
+
+* `|` denotes a pipe. The output of the previous query acts as ab input to the next query.
+* `$-` refers to the input stream.
+* The second approach adopts a user-defined variable `$var`. The scope of this variable is within the compound statement.
+
+## Batch Inserting
+
+To insert multiple data, you can put all the DDL (data definition language) statements in a `.text` file as follows.
 
 ```ngql
-DROP SPACE test; -- Delete useless space
-CREATE SPACE test(partition_num=1024, replica_factor=1);
-USE test;
-CREATE TAG course(name string, credits int);
-CREATE TAG building(name string);
-CREATE TAG student(name string, age int, gender string);
-CREATE EDGE like(likeness double);
-CREATE EDGE select(grade int);
+CREATE SPACE nba(partition_num=1024, replica_factor=1);
+USE nba;
+CREATE TAG player(name string, age int);
+CREATE TAG team(name string);
+CREATE EDGE follow(degree int);
+CREATE EDGE serve(start_year int, end_year int);
 ```
 
-Then batch write to console
+* IF you install **Nebula Graph** by compiling the source code, you can batch write to console by the following command:
 
 ```bash
 cat  schema.ngql | ./bin/nebula -u user -p password
 ```
 
-Likewise, you can put hundreds or thousands DML statements in a data.ngql to insert. If you have millions of records to insert, please use the csv importer tool (or sst ingest tool).
+* IF you install **Nebula Graph** by docker-compose, you can batch write to console by the following command:
+
+```bash
+cat nba.text | docker run --rm -i --network=host vesoft/nebula-console:nightly --addr=127.0.0.1 --port=32797
+```
+
+**Note**: You must change the IP address and the port number to yours.
+
+Likewise, you can put hundreds or thousands DML statements in a `data.text` file to insert data. If you have millions of records to insert, it is recommended to use the csv importer tool (or sst ingest tool).
+
+
```

Comment:
The nba.ngql can be downloaded [here](https://nebula-graph.oss-accelerate.aliyuncs.com/doc/nba.ngql)

https://nebula-graph.oss-accelerate.aliyuncs.com/doc/nba.ngql

`OSS` is our cloud-storage